build/helper.ts

```diff
@@ -19,6 +19,14 @@ const reDocUrl: InstallOptions['reDocUrl'] = (fileName, header) => {
   return docs + fileName + (_header ? `#${_header}` : '')
 }
 
+const reWebTypesSource: InstallOptions['reWebTypesSource'] = (title) => {
+  const symbol = `El${title.replace(/-/, ' ').replace(/^\w|\s+\w/g, (item) => {
```

Comment:
```suggestion
  const symbol = `El${title.replace('-', ' ').replaceAll(/^\w|\s+\w/, (item) => {
```

Reply:
When using regular, the replace is the same as the replaceAll. For easy reading?

Reply:
Yeah. I think `replaceAll` is better.